src/pages/auditing/reportUpload/reportUpload.schema.tsx

```diff
@@ -56,24 +56,14 @@ export const reportUploadSchema = yup.object().shape({
       /^(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})$/,
       "Please verify the characters entered"
```

Comment:
The error message is not really useful, maybe something like:
```suggestion
      "Please enter a valid URL."
```

Reply:
Done